### ISSUE TITLE
Upgrade to librdkafka 0.9 with support for high-level consumer

### ISSUE THREAD

**Issue by thijsc** [buoyant-data/hermann#139]
`librdkafka` 0.9 introduces support for features such as the high-level consumer and SSL connections. It would be great if Hermann would leverage this.

0.9 has been out for quite some time. Is there any technical reason holding this back? Or is there no interest in supporting this? If there's interest in potentially accepting a pull for this I'd be happy to take a stab at it.

**Comment by rtyler** [comment]
@thijsc no reason other than nobody's submitted a pull request :wink:

**Comment by thijsc** [comment]
Did a quick trial run and. I had to disable SASL and the logging function was deprecated. Everything mostly seems to work with 0.9.2. I don't think SSL needs any special support, it's a matter of setting the right configuration.

Looking at the consumer class I'm not sure how to proceed. I noticed that it does support the high-level consumer in Java. But if I would change this to high-level for MRI too that behaviour would completely change.

I guess the current consumer would have to be backwards compatible? Maybe it would be wiser to introduce a `HighLevelConsumer` alongside the current simple consumer?
